### PR TITLE
LEARNER-4122 Android - Verify 'Accessibility ID' is properly set against each element - for New Logistration Screen

### PR DESCRIPTION
**Pull request by nedo**
* Implemented a wait_for_android_activity_to_load() function
* all element locations have proper locators based on given ids

### CODE REVIEW

**Comment by nedo** [comment]
@raeeschachar PR is ready for review

cc @staubina

**Comment by nedo** [comment]
@raeeschachar I've rebased this PR with master, its ready for review

cc @staubina

File: android/tests/test_android_new_logistration.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from android.pages.android_new_logistration import AndroidNewLogistration
 from common import strings
 
+
```
Comment (raeeschachar) [inline android/tests/test_android_new_logistration.py]: Line 24, multiple spaces after operator 
Non Ascii characters in file, declare encoding

File: common/globals.py [hunk 2]
```diff
@@ -7,18 +7,19 @@
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions
 from selenium.webdriver.support.wait import WebDriverWait
-from input_data import InputData
 from common import strings
+from input_data import InputData
+
 
 
 class Globals:
```
Comment (raeeschachar) [inline common/globals.py]: line 15, too many blank lines 
line 126, visibility spellings
line 155, invisibility spellings
line 212, too many blank lines

File: common/globals.py [hunk 3]
```diff
@@ -184,5 +186,60 @@ def scroll_screen(self, driver, from_element, to_element):
...
+                target_activity,
+                any_exception,
+                sys.exc_info()[0]
+
+            ))
+
+            return False
+
+
+class wait_for_activity(object):
```
Comment (raeeschachar) [inline common/globals.py]: Class name should have CamelCase convention

File: common/globals.py [hunk 4]
```diff
@@ -184,5 +186,60 @@ def scroll_screen(self, driver, from_element, to_element):
...
+
+  def __call__(self, driver):
+      self.driver = driver
+      if self.driver.current_activity == self.target_activity:
+          self.log.info('on {} '.format(self.target_activity))
+          return self.driver.current_activity
+
+      else:
+        self.log.error('{} - {} '.format(self.target_activity, strings.ERROR_SCREEN_NOT_LOADED))
+        return False
```
Comment (raeeschachar) [inline common/globals.py]: Empty line at the end of file

File: common/globals.py (hunk 3 above)
Comment (raeeschachar) [inline common/globals.py]: This complete class needs review ... indentation issues, CamelCase convention issue, empty line at the end of file issue

**Comment by nedo** [comment]
[WIP] Rebased with master, still in progress

File: android/pages/android_new_logistration.py [hunk 5]
```diff
@@ -1,6 +1,7 @@
 """
```
[the remark this answers is not in the document]
Comment (nedo) [reply android/pages/android_new_logistration.py]: done

File: android/tests/test_android_new_logistration.py (hunk 1 above)
Comment (nedo) [reply android/tests/test_android_new_logistration.py]: done

File: common/globals.py (hunk 2 above)
Comment (nedo) [reply common/globals.py]: done

File: common/globals.py (hunk 3 above)
Comment (nedo) [reply common/globals.py]: done

Comment (nedo) [reply common/globals.py]: done

**Comment by nedo** [comment]
@raeeschachar PR is ready for another review

cc @staubina

File: conftest.py [hunk 6]
```diff
@@ -2,18 +2,21 @@
...
+from common import strings
+from common.globals import Globals
+from input_data import InputData
 from ios.pages.ios_login import IosLogin
-from android.pages.android_login import AndroidLogin
+from ios.pages.ios_new_logistration import IosNewLogistration
+
 
 
 @pytest.fixture(scope="session")
```
Comment (raeeschachar) [inline conftest.py]: Too many blank lines

File: android/pages/android_login.py [hunk 7]
```diff
@@ -1,10 +1,10 @@
 """
     Login Page Module
```
[the remark this answers is not in the document]
Comment (nedo) [reply android/pages/android_login.py]: I've noticed this point and this is done in PR#12 already in your plate

**Comment by nedo** [comment]
@staubina PR is ready for your Pass

File: android/pages/android_new_logistration.py [hunk 8]
```diff
@@ -98,10 +97,13 @@ def load_login_screen(self):
         """
 
         self.get_signin_button().click()
-        self.get_screen_title_textview()
-        self.log.info(self.driver.current_activity)
 
-        return self.driver.current_activity
+        current_activity = self.global_contents.wait_for_android_activity_to_load(
```
[the remark this answers is not in the document]
Comment (nedo) [reply android/pages/android_new_logistration.py]: done, this missing point as it is implemented everywhere else.

**Pull request merged**